### DISCUSSION TITLE
Github copilot using wrong package version

### DISCUSSION THREAD

**Discussion by Asynchronite** [community/community#116535] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/60348001/398f9236-c609-419d-a897-a1ec0c8a11db)
Github copilot can only answer questions for discord.js up to version 12.3.1, and yet the newest discord.js version is v14.14.1, how is that possible?

**Comment by kuciapakrystian** [comment]
Github copilot is a machine learning AI that uses a dataset of code examples and documentation, and it may not have been trained on the latest versions of all libraries and frameworks. In this case, it seems that Github copilot has not been updated to support discord.js version 14.14.1, so it can only provide assistance for up to version 12.3.1. Since copilot is constantly evolving, it's possible that future updates may include support for newer versions of discord.js. :)
